synapse/models/inet.py

```diff
@@ -344,28 +345,41 @@ def getModelDefs(self):
                     )),
 
                     ('inet:fqdn', {}, (
+                        # FIXME uncomment times when temporal is done
```

Comment:
These should just end up as ('time', {'ismin':True}) etc ( no more int:min int:max time:min time:max )